lib/utils/injectRefreshEntry.js

```diff
@@ -38,13 +37,11 @@ function injectRefreshEntry(originalEntry, options, compiler) {
   let resourceQuery = {};
 
   if (compiler && compiler.options && compiler.options.devServer) {
-    resourceQuery.domain = createDomain(compiler.options.devServer);
-    if (compiler.options.devServer.sockHost)
-      resourceQuery.sockHost = compiler.options.devServer.sockHost;
-    if (compiler.options.devServer.sockPath)
-      resourceQuery.sockPath = compiler.options.devServer.sockPath;
-    if (compiler.options.devServer.sockPort)
-      resourceQuery.sockPath = compiler.options.devServer.sockPort;
```

Comment:
both of these were writing into `sockPath`

Reply:
good catch